_posts/2016-04-11-growing-developers-at-greenfield.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: post
+title: "Growing Developers at Greenfield"
+description: "Advice for junior developers getting started in their careers"
+category:
+tags: [lessons, advice, career]
+author: tom_chen
+---
+{% include JB/setup %}
+
+![tree]({{ BASE_PATH
+}}/assets/images/growing-developers-at-greenfield/tree.jpeg)
+
+Before I was a developer at Greenfield, I spent several years working in
+academia. After deciding to make the leap into full-time software development
+and completing [Launch Academy](https://www.launchacademy.com/) last year, I was
+anxious to join a company where I could work with and learn from experienced
+developers who were shipping high quality code. When I arrived at Greenfield, I
+was happy to find a team that was not only welcoming and supportive but also
+active in the open source community, passionate about best practices, and as
+excited about building awesome software as I am. Having been at Greenfield for a
+few months now, I'd like to share a couple of lessons I’ve learned that might be
+valuable to others just getting started in their software careers.
+
+### Think Process, Not Product
```

Comment:
"Think Winning, Not Scoring"? -- sports analogy
"Think Set, Not Game"? -- tennis
"Think Strategy, Not Tactic"?

Maybe just "Focus on the Process" works too